### DISCUSSION TITLE
Codaqui na Conferência do DevPR 2023 - CODAQUI.dev

### DISCUSSION THREAD

**Discussion by giscus** [codaqui/institucional#112] [Blog]
# Codaqui na Conferência do DevPR 2023 - CODAQUI.dev

Queremos democratizar o aprendizado tecnológico, e aproximar novas gerações do conteúdo tecnico.

https://www.codaqui.dev/blog/2023/01/05/codaqui-na-confer%C3%AAncia-do-devpr-2023/

**Comment by endersonmenezes** [comment]
Uhuuuuuuuuuuuul, parabéns a toda organização do evento, por estarem se esforçando para trazer toda essa banca de conteúdo para gente, em especial um parabéns ao @guiseek por sua dedicação.
